.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,77 @@
+name: Bug report for the Element iOS app
+description: Report any issues that you have found with the Element app. Please [check open issues](https://github.com/vector-im/element-ios/issues) first, in case it has already been reported.
+labels: [T-Defect]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        Please report security issues by email to security@matrix.org
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Steps to reproduce
+      description: Please attach screenshots, videos or logs if you can.
+      placeholder: Tell us what you see!
+      value: |
+        1. Where are you starting? What can you see?
+        2. What do you click?
+        3. More steps…
+    validations:
+      required: true
+  - type: textarea
+    id: expected-result
+    attributes:
+      label: What did you expect?
+      placeholder: Tell us what you expected to happen in as much detail as you can.
+    validations:
+      required: true
+  - type: textarea
+    id: actual-result
+    attributes:
+      label: What happened?
+      placeholder: Tell us what went wrong
+    validations:
+      required: true
+  - type: input
+    id: device
+    attributes:
+      label: Your phone model
+      placeholder: e.g. iPhoneX
+    validations:
+      required: false
+  - type: input
+    id: os
+    attributes:
+      label: Operating system version
+      placeholder: e.g. iOS8.1, under "software version"
```

```suggestion
      placeholder: e.g. iOS14.7.1, under "software version"
```

to look more up-to-date